mautic/mautic
# Sync boolean by value

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
At the moment we can customize custom contact field options of boolean type:
![image](https://user-images.githubusercontent.com/462477/80352226-dab2f800-8873-11ea-8094-02cd8004f15d.png)
If you created form with checkbox field and want it match  to contact's bool field Mautic expect value yes or 1 or true.

This PR added options sync this fields by values

![image](https://user-images.githubusercontent.com/462477/80352396-1f3e9380-8874-11ea-9f02-d1e76ff6443e.png)

For people without english language can help manage that syncing. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Load up [this PR](https://mautibox.com)
2. Create custom contact bool type with Unconfirmed/Confirmed values of bool type
![image](https://user-images.githubusercontent.com/462477/80352686-89efcf00-8874-11ea-8558-9ed1d1f87ddb.png)
3. The create form with checkbox group type and add to properties tab Confirmed as values and map it to created boolean type
![image](https://user-images.githubusercontent.com/462477/80352620-6dec2d80-8874-11ea-96b7-bab565c149d8.png)
4. Send form with checked checkbox field and check contact's bool type fields in Mautic. Should be red (Unconfirmed),but expect Confirmed


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps, contact't fields should be Confirmed after sent form
